DeAceroDLabs/marketplace-frontend
# feat: [DMP-603] Added docker compose.

## What is done

[[DMP-603] Added docker compose](https://deacero.atlassian.net/browse/DMP-603)

## How is done

- Added a `docker compose` file and updated setupProxy so that it used the container name from back instead of localhost when running docker.

## How to test

1. Please approve/see first the pr DMP-606 on the back.
2. The 2 back containers should be running (check with `docker ps`). Or check with docker desktop.
3. Run the project via `docker-compose up --build` or `npm run docker:run` This command is to make sure images are updated (no need to remove images or containers anymore).
4. Open localhost:8300, everything should work.
